app/views/conferences/index.html.slim

```diff
@@ -21,6 +22,7 @@ table.table.table-striped.table-bordered.table-condensed.conferences
       td = format_conference_date(conference.starts_on, conference.ends_on)
       td = conference.attendees.any? ? links_to_attendances(conference).join(', ').html_safe : '-'
       td = conference.tickets_left
+      td = conference.lightningtalkslots
```

Comment:
I've just realised: this would display a `true` / `false` (or nothing at all) to the user. Could you improve on that by displaying a »yes« (or »no«, respectively)?